components/Slider.tsx

```diff
@@ -1,25 +1,25 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
```

Comment:
It seems that you have done nothing special in this file except formatting the code. Revert the changes.